### DISCUSSION TITLE
科技爱好者周刊（第 323 期）：技术公司的口号比拼 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#45] [General]
# 科技爱好者周刊（第 323 期）：技术公司的口号比拼 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-323

**Comment by eq1024** [comment]
一直在用你的vercel看日志,继续保持❤

**Reply by plantree** [reply]
哈哈，感谢支持！有想法可以多交流~
